test/system/000-setup.bats

```diff
@@ -15,6 +15,18 @@
 # limitations under the License.
 #
 
+if ! [ -f test/system/libs/bats-assert/load.bash ]; then
```

Comment:
It might be better to move this to the [setup_suite](https://bats-core.readthedocs.io/en/stable/writing-tests.html) function.

Currently, it fails as:
```
$ TOOLBOX=/opt/bin/toolbox bats ./test/system
000-setup.bats
 ✗ setup_file failed
   (in test file test/system/000-setup.bats, line 21)
     `exit 1' failed with status 0
   test/system/libs/bats-assert/load.bash is missing
   Forgot to run 'git submodule init' and 'git submodule update' ?
001-version.bats
 ✗ setup_file failed
   bats_load_safe: Could not find '/tmp/toolbox/test/system/libs/bats-support/load'[.bash]
002-help.bats
 ✗ setup_file failed
   bats_load_safe: Could not find '/tmp/toolbox/test/system/libs/bats-support/load'[.bash]
101-create.bats
 ✗ setup_file failed
   bats_load_safe: Could not find '/tmp/toolbox/test/system/libs/bats-support/load'[.bash]
102-list.bats
 ✗ setup_file failed
   bats_load_safe: Could not find '/tmp/toolbox/test/system/libs/bats-support/load'[.bash]
103-container.bats
 ✗ setup_file failed
   bats_load_safe: Could not find '/tmp/toolbox/test/system/libs/bats-support/load'[.bash]
104-run.bats
 ✗ setup_file failed
   bats_load_safe: Could not find '/tmp/toolbox/test/system/libs/bats-support/load'[.bash]
105-enter.bats
 ✗ setup_file failed
   bats_load_safe: Could not find '/tmp/toolbox/test/system/libs/bats-support/load'[.bash]
106-rm.bats
 ✗ setup_file failed
   bats_load_safe: Could not find '/tmp/toolbox/test/system/libs/bats-support/load'[.bash]
107-rmi.bats
 ✗ setup_file failed
   bats_load_safe: Could not find '/tmp/toolbox/test/system/libs/bats-support/load'[.bash]
108-completion.bats
 ✗ setup_file failed
   bats_load_safe: Could not find '/tmp/toolbox/test/system/libs/bats-support/load'[.bash]
999-teardown.bats
 ✗ setup_file failed
   (in test file test/system/999-teardown.bats, line 18)
     `load 'libs/helpers'' failed with status 0
   bats_load_safe: Could not find '/tmp/toolbox/test/system/libs/bats-support/load'[.bash]
   bats warning: Executed 12 instead of expected 193 tests

193 tests, 12 failures, 181 not run
```

With `setup_suite`, it seems to fail as:
```
$ TOOLBOX=/opt/bin/toolbox bats ./test/system
 ✗ setup_suite
   Forgot to run 'git submodule init' and 'git submodule update' ?
   (from function `setup_suite' in test file test/system/setup_suite.bash, line 5)
     `return 1' failed
   bats warning: Executed 1 instead of expected 193 tests

193 tests, 1 failure, 192 not run
```

... which is cleaner.  Although, I have no idea why it doesn't show the `... load.bash is missing` error message.

In fact, the entire `000-setup.bats` should probably go into `setup_suite`.

Reply:
Our Git [history](https://github.com/containers/toolbox/commit/54a2ca1ead343c3db2c7443cc58b9d9e896d3c46) indicates that our `000-setup.bats` and `999-teardown.bats` were added before Bats started to offer `setup_suite` and `teardown_suite`.  Those [became](https://github.com/bats-core/bats-core/commit/fb467ec3f04e322aee335c92c374158edfff0344) a thing in Bats 1.7.0, which we now require anyway.

Reply:
The move to `setup_suite` and `teardown_suite` was done in https://github.com/containers/toolbox/pull/1317

Reply:
>  I have no idea why it doesn't show the `... load.bash is missing` error message.

Moving the error output to `setup_suite` fixes the problem of the missing error message.  We [shouldn't](https://bats-core.readthedocs.io/en/stable/writing-tests.html) be printing outside the `@test`, `setup*` or `teardown*` functions in general.